QuadPaySDK/PaymentWidget/PaymentWidgetHeaderText.swift

```diff
@@ -26,6 +26,9 @@ public final class PaymentWidgetHeaderText: UIView {
     var initialMinModal : String = ""
     var minModal : String?
     
+    //var initialHasFees : String = ""
+    var hasFees : String?
```

Comment:
Should be a `boolean` defaulting to `false` so we don't need to use an optional value.

Delete comment above `initialHasFees`